requestmanager/asyncloader/unverifiedblockstore/unverifiedblockstore.go

```diff
@@ -55,6 +63,9 @@ func (ubs *UnverifiedBlockStore) VerifyBlock(lnk ipld.Link) ([]byte, error) {
 		return nil, fmt.Errorf("Block not found")
 	}
 	delete(ubs.inMemoryBlocks, lnk)
+	ubs.dataSize = ubs.dataSize - uint64(len(data))
+
+
```

Comment:
Two blank lines doesn't pass gofmt.
```suggestion
	ubs.dataSize = ubs.dataSize - uint64(len(data))

```